pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -125,9 +127,9 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 		}
 
 		clusterImage := cephCluster.Spec.CephVersion.Image
-		cephVersion, ok := getImageVersion(clusterImage)
-		if !ok {
-			logger.Warningf("ceph version not found for image %q used by cluster %q", clusterImage, cephCluster.Name)
+		cephVersion, err := getImageVersion(cephCluster)
+		if err != nil {
+			logger.Warningf("ceph version not found for image %q used by cluster %q: %v", clusterImage, cephCluster.Name, err)
```

Comment:
Errors should be separated from additional text by periods, not colons.


```suggestion
			logger.Warningf("ceph version not found for image %q used by cluster %q. %v", clusterImage, cephCluster.Name, err)
```